study-builder/studies/osteo/study.conf

```diff
@@ -2189,6 +2189,83 @@
       "dispatchToHousekeeping": true,
       "order": 1
     },
+    ## age-up next steps email
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
```

Comment:
This should use `INVITATION_EMAIL` type so we embedded the invitation id in the email. Same deal for the reminder email events.